dotnet-projects/StyleCop/StyleCop/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace StyleCop.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
+    {
+        private static readonly string[] Summaries = new[]
+        {
+        "Freezing", "Bracing", "Chilly", "Cool", "Mild",
+         "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+        };
```

Comment:
Formatting is not right : 
```suggestion
        private static readonly string[] Summaries = new[]
        {
            "Freezing", "Bracing", "Chilly", "Cool", "Mild",
            "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
        };

```

Reply:
I did that on purpose to suppress that rule in GlobalSupression.cs file :)

Reply:
Hey @amuslic, Oh right. my bad...
Thanks :)